cmd/rpm2tar.go

```diff
@@ -62,7 +63,7 @@ func NewRpm2TarCmd() *cobra.Command {
 								Typeflag: tar.TypeSymlink,
 								Name:     k,
 								Linkname: v,
-								Mode:     0777,
+								Mode:     0o777,
```

Comment:
Hm, this is just a symlink, 0777 should be good here. The actual permission is then defined by the symlink target. Did you have any issues with that?

Reply:
This was a formatting change done by my formatter (gofumpt). Both are octal numbers: https://go.dev/ref/spec#Integer_literals
I can revert if you want!

Reply:
ack!